src/MemUtils_CUDA.cxx

```diff
@@ -73,16 +74,30 @@ namespace RAJA
 // Static array used to keep track of which unique ids
 // for CUDA reduction objects are used and which are not.
 //
-static bool cuda_reduction_id_used[RAJA_MAX_REDUCE_VARS];
+static bool s_cuda_reduction_id_used[RAJA_MAX_REDUCE_VARS];
 
 //
 // Pointer to hold shared managed memory block for RAJA-Cuda reductions.
 //
-CudaReductionBlockDataType* s_cuda_reduction_mem_block = 0;
+static CudaReductionDummyBlockType* s_cuda_reduction_mem_block = 0;
```

Comment:
I agree; so I'll have to ping Jason to see why they were changed, as it could be build related.